src/note.ts

```diff
@@ -756,18 +756,17 @@ export class Rest extends GeneralNote {
      */
     vexflowNote(options): Vex.Flow.StaveNote {
         let keyLine = 'b/4';
-        if (this.duration.type === 'whole') {
-            if (
-                this.activeSite !== undefined
-                && this.activeSite.renderOptions.staffLines !== 1
-            ) {
-                keyLine = 'd/5';
-            }
+        const activeSiteSingleLine = (
+            this.activeSite !== undefined
+            && this.activeSite.renderOptions.staffLines === 1
+        );
+        if (this.duration.type === 'whole' && !activeSiteSingleLine) {
+            keyLine = 'd/5';
         }
-        if (this.lineShift !== undefined) {
+        if (this.stepShift !== undefined) {
```

Comment:
Next PR just changed this to !== 0 and it'll speed up not having to create a new pitch for every rest.

Reply:
Done in https://github.com/cuthbertLab/music21j/pull/151